Doc/library/doctest.rst

```diff
@@ -300,33 +301,49 @@ their contained methods and nested classes.
 How are Docstring Examples Recognized?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The :mod:`doctest` module treats any line beginning with ``>>>`` as an 
+example to be tested. 
+Following lines which begin with ``...`` continue the example. 
```

Comment:
I'm using 'example' because I think that's what this document chose as the term for this line-beginning-with-`>>>` thing. I think 'test' would be a clearer term, but then the whole document should be changed to use 'test'. Let's choose one and make the document consistent.